packs/atlassian.yml

```diff
@@ -4,6 +4,7 @@ Description: Group of all Atlassian detections
 PackDefinition:
   IDs:
     - Atlassian.User.LoggedInAsUser
+    - Confluence.0DayIPs
```

Comment:
These IOCs were last updated # 2022-06-02 Confluence 0-Day IOCs: This rule probably needs to be deprecated.